ballerina/kafka_records.bal

```diff
@@ -198,6 +198,9 @@ public type TopicPartition record {|
     int partition;
 |};
 
+type ByteHeaderValue byte[]|byte[][];
+type AnydataHeaderValue ByteHeaderValue|string|string[];
```

Comment:
Wouldn't the name `AnydataHeaderValue` suggest that the type has something to do with ballerina `anydata` type?